<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #212529;
QPushButton
{
border: 0px solid rgb(249, 167, 43);
border-bottom-width: 2px;
border-radius: px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane {
    background: transparent;
    border:0;
}

QTabBar::tab
{
	background-color: rgba(42, 46, 50,0);
	color:#ffffff;
	width:100px;
	font-size:16px;
	font-family:Calibri;

}

QTabBar::tab:hover
{
	background-color:rgb(50, 53, 57);
		color:rgb(249, 167, 43);
}

QTabBar::tab:focus
{
	background-color: rgb(59, 62, 66);
	border-style:solid;
	border-left-width:1px;
	border-bottom-width:1px;
	border-color: rgb(249, 167, 43);
	color:rgb(249, 167, 43);
}
QTabBar::tab:selected
{
	background-color: rgb(59, 62, 66);
	border-style:solid;
	border-left-width:1px;
	border-bottom-width:1px;
	border-color:   rgb(249, 167, 43);
	color:  rgb(249, 167, 43);
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>sites</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>801</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane {
    background: transparent;
    border:0;
}

QTabBar::tab
{
	background-color: rgba(42, 46, 50,0);
	color:#ffffff;
	width:100px;
	font-size:16px;
	font-family:Calibri;

}

QTabBar::tab:hover
{
	background-color:rgb(50, 53, 57);
		color:rgb(249, 167, 43);
}

QTabBar::tab:focus
{
	background-color: rgb(59, 62, 66);
	border-style:solid;
	border-left-width:1px;
	border-bottom-width:1px;
	border-color: rgb(249, 167, 43);
	color:rgb(249, 167, 43);
}
QTabBar::tab:selected
{
	background-color: rgb(59, 62, 66);
	border-style:solid;
	border-left-width:1px;
	border-bottom-width:1px;
	border-color:   rgb(249, 167, 43);
	color:  rgb(249, 167, 43);
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>160</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>lien:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>220</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>280</y>
          <width>221</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color:rgb(50, 53, 57);
	border-style:solid;
	border-radius:7px;
	border-width:1px;
border-color: rgb(50, 53, 57);
	color:#ffffff;
	font-size:16px;
font-family:Calibri;
}

QPushButton:hover
{
	background-color: #00000000;
	border-color: rgb(30, 227, 0);
	/*color:rgb(30, 227, 0);*/
}

QPushButton:focus
{
	background-color:  rgb(30, 227, 0);
	border-color: rgba(30, 227, 0,0);
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_etat">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>210</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px solid rgb(249, 167, 43);
border-bottom-width: 2px;
border-radius: px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_num">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>90</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px solid rgb(249, 167, 43);
border-bottom-width: 2px;
border-radius: px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>100</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_eclair">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>150</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px solid rgb(249, 167, 43);
border-bottom-width: 2px;
border-radius: px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_ajout">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>250</y>
          <width>221</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color:rgb(50, 53, 57);
	border-style:solid;
	border-radius:7px;
	border-width:1px;
border-color: rgb(50, 53, 57);
	color:#ffffff;
	font-size:16px;
font-family:Calibri;
}

QPushButton:hover
{
	background-color: #00000000;
	border-color: rgb(30, 227, 0);
	/*color:rgb(30, 227, 0);*/
}

QPushButton:focus
{
	background-color:  rgb(30, 227, 0);
	border-color: rgba(30, 227, 0,0);
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tab_chambre">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>791</width>
          <height>501</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(249, 167, 43);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px solid rgb(249, 167, 43);
border-bottom-width: 2px;
border-radius: px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color:rgb(50, 53, 57);
	border-style:solid;
	border-radius:7px;
	border-width:1px;
border-color: rgb(50, 53, 57);
	color:#ffffff;
	font-size:16px;
font-family:Calibri;
}

QPushButton:hover
{
	background-color: #00000000;
	border-color: rgb(30, 227, 0);
	/*color:rgb(30, 227, 0);*/
}

QPushButton:focus
{
	background-color:  rgb(30, 227, 0);
	border-color: rgba(30, 227, 0,0);
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>30</y>
          <width>73</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <item>
         <property name="text">
          <string>ID_SITE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LIEN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DESCRIPTION</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>30</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color:rgb(50, 53, 57);
	border-style:solid;
	border-radius:7px;
	border-width:1px;
border-color: rgb(50, 53, 57);
	color:#ffffff;
	font-size:16px;
font-family:Calibri;
}

QPushButton:hover
{
	background-color: #00000000;
	border-color: rgb(30, 227, 0);
	/*color:rgb(30, 227, 0);*/
}

QPushButton:focus
{
	background-color:  rgb(30, 227, 0);
	border-color: rgba(30, 227, 0,0);
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Trier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>30</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color:rgb(50, 53, 57);
	border-style:solid;
	border-radius:7px;
	border-width:1px;
border-color: rgb(50, 53, 57);
	color:#ffffff;
	font-size:16px;
font-family:Calibri;
}

QPushButton:hover
{
	background-color: #00000000;
	border-color: rgb(30, 227, 0);
	/*color:rgb(30, 227, 0);*/
}

QPushButton:focus
{
	background-color:  rgb(30, 227, 0);
	border-color: rgba(30, 227, 0,0);
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>export PDF</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_supp">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>30</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color:rgb(50, 53, 57);
	border-style:solid;
	border-radius:7px;
	border-width:1px;
border-color: rgb(50, 53, 57);
	color:#ffffff;
	font-size:16px;
font-family:Calibri;
}

QPushButton:hover
{
	background-color: #00000000;
	border-color: rgb(30, 227, 0);
	/*color:rgb(30, 227, 0);*/
}

QPushButton:focus
{
	background-color:  rgb(30, 227, 0);
	border-color: rgba(30, 227, 0,0);
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Statistiques</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>90</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(249, 167, 43);</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>enfant</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>811</width>
        <height>551</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Ajouter </string>
       </attribute>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>190</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>age:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_etat_2">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>100</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px solid rgb(249, 167, 43);
border-bottom-width: 2px;
border-radius: px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>192</x>
          <y>230</y>
          <width>231</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color:rgb(50, 53, 57);
	border-style:solid;
	border-radius:7px;
	border-width:1px;
border-color: rgb(50, 53, 57);
	color:#ffffff;
	font-size:16px;
font-family:Calibri;
}

QPushButton:hover
{
	background-color: #00000000;
	border-color: rgb(30, 227, 0);
	/*color:rgb(30, 227, 0);*/
}

QPushButton:focus
{
	background-color:  rgb(30, 227, 0);
	border-color: rgba(30, 227, 0,0);
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>50</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>140</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>prenom:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_num_2">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>50</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px solid rgb(249, 167, 43);
border-bottom-width: 2px;
border-radius: px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_prenom">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>140</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px solid rgb(249, 167, 43);
border-bottom-width: 2px;
border-radius: px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_ajout_2">
        <property name="geometry">
         <rect>
          <x>192</x>
          <y>260</y>
          <width>231</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color:rgb(50, 53, 57);
	border-style:solid;
	border-radius:7px;
	border-width:1px;
border-color: rgb(50, 53, 57);
	color:#ffffff;
	font-size:16px;
font-family:Calibri;
}

QPushButton:hover
{
	background-color: #00000000;
	border-color: rgb(30, 227, 0);
	/*color:rgb(30, 227, 0);*/
}

QPushButton:focus
{
	background-color:  rgb(30, 227, 0);
	border-color: rgba(30, 227, 0,0);
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_age">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>190</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px solid rgb(249, 167, 43);
border-bottom-width: 2px;
border-radius: px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>100</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>nom:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tab_lampe">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>110</y>
          <width>451</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px solid rgb(249, 167, 43);
border-bottom-width: 2px;
border-radius: px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>70</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color:rgb(50, 53, 57);
	border-style:solid;
	border-radius:7px;
	border-width:1px;
border-color: rgb(50, 53, 57);
	color:#ffffff;
	font-size:16px;
font-family:Calibri;
}

QPushButton:hover
{
	background-color: #00000000;
	border-color: rgb(30, 227, 0);
	/*color:rgb(30, 227, 0);*/
}

QPushButton:focus
{
	background-color:  rgb(30, 227, 0);
	border-color: rgba(30, 227, 0,0);
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>430</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color:rgb(50, 53, 57);
	border-style:solid;
	border-radius:7px;
	border-width:1px;
border-color: rgb(50, 53, 57);
	color:#ffffff;
	font-size:16px;
font-family:Calibri;
}

QPushButton:hover
{
	background-color: #00000000;
	border-color: rgb(30, 227, 0);
	/*color:rgb(30, 227, 0);*/
}

QPushButton:focus
{
	background-color:  rgb(30, 227, 0);
	border-color: rgba(30, 227, 0,0);
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>export PDF</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_supp_2">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>110</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color:rgb(50, 53, 57);
	border-style:solid;
	border-radius:7px;
	border-width:1px;
border-color: rgb(50, 53, 57);
	color:#ffffff;
	font-size:16px;
font-family:Calibri;
}

QPushButton:hover
{
	background-color: #00000000;
	border-color: rgb(30, 227, 0);
	/*color:rgb(30, 227, 0);*/
}

QPushButton:focus
{
	background-color:  rgb(30, 227, 0);
	border-color: rgba(30, 227, 0,0);
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>160</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color:rgb(50, 53, 57);
	border-style:solid;
	border-radius:7px;
	border-width:1px;
border-color: rgb(50, 53, 57);
	color:#ffffff;
	font-size:16px;
font-family:Calibri;
}

QPushButton:hover
{
	background-color: #00000000;
	border-color: rgb(30, 227, 0);
	/*color:rgb(30, 227, 0);*/
}

QPushButton:focus
{
	background-color:  rgb(30, 227, 0);
	border-color: rgba(30, 227, 0,0);
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
